src/sentry/event_manager.py

```diff
@@ -2044,49 +2045,46 @@ def _process_existing_aggregate(
 
 def _get_severity_score(event: Event) -> float | None:
     op = "event_manager._get_severity_score"
-    logger_data = {"event_id": event.data["event_id"], "op": op}
+    title = ErrorEvent().compute_title(dict(event.get_event_metadata()))
+    logger_data = {"event_id": event.data["event_id"], "op": op, "event_title": title}
     severity = None
 
-    metadata = event.get_event_metadata()
-    error_type = metadata.get("type")
-    error_msg = metadata.get("value")
-    message = ""
-    if error_type:
-        message = error_type if not error_msg else f"{error_type}: {error_msg}"
-
-    if message:
-        logger_data["event_message"] = message
-        with metrics.timer(op):
-            with sentry_sdk.start_span(op=op):
-                try:
-                    response = severity_connection_pool.urlopen(
-                        "POST",
-                        "/issues/severity-score",
-                        body=json.dumps({"message": message or "<unknown event>"}),
-                        headers={"content-type": "application/json;charset=utf-8"},
-                    )
-                    severity = json.loads(response.data).get("severity")
-                except MaxRetryError as e:
-                    logger.warning(
-                        f"Unable to get severity score from microservice after {SEVERITY_DETECTION_RETRIES} retr{'ies' if SEVERITY_DETECTION_RETRIES >1 else 'y'}. Got MaxRetryError caused by: {repr(e.reason)}.",
-                        extra=logger_data,
-                    )
-                except Exception as e:
-                    logger.warning(
-                        f"Unable to get severity score from microservice. Got: {repr(e)}.",
-                        extra=logger_data,
-                    )
-                else:
-                    logger.info(
-                        f"Got severity score of {severity} for event {event.data['event_id']}",
-                        extra=logger_data,
-                    )
-    else:
-        logger_data.update({"error_type": error_type, "error_msg": error_msg})
+    if title == "<unknown>":
         logger.warning(
-            "Unable to get severity score because event has no message",
+            "Unable to get severity score because event has title '<unknown>'",
```

Comment:
Just an aside - I wonder if we should have some sort of default severity score we give to issues like these, or if the severity model can handle unknown event titles

Reply:
In theory, these issues _should_ be few and quite far between. We could query actual event data to be sure, but I believe that any error event generated by an SDK (either automatically or because the user calls `captureException`) should have enough information (error type and value, or, failing that, erroring function) to come up with a title other than `<unknown>`. That said, we actually should talk to @trillville about this, because as soon as I add in more payload data, we'll have to decide if lack of a title is still disqualifying or if the rest of the data (currently `has_stacktrace`, `handled`, and `level_is_error`) is still enough to generate a score.

That's Next-PR-Katie's problem, though. 🙂

Reply:
Update: Tillman confirmed that not having a usable message is disqualifying, so kept that early out.